assets/js/googlesitekit/datastore/site/urls.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * `core/site` data store: urls.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { getLocale, getLocaleLanguage } from '../../../util';
+
+export const initialState = {};
+
+export const actions = {};
+
+export const controls = {};
+
+export function reducer( state ) {
+	return state;
+}
+
+export const resolvers = {};
```

Comment:
Since these are all empty we can actually omit them, our `combineStores` utility will handle missing arguments.

I don't think there's much use in having this boilerplate here. 🤷🏻‍♂️

Reply:
Ok, removed.